packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { string, node } from 'prop-types';
+import styled from 'styled-components';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import {
+  GEL_MARGIN_BELOW_400PX,
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_LONG_PRIMER } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const StyledGuidance = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_LONG_PRIMER};
+`;
+
+const GuidanceWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-color: rgba(34, 34, 34, 0.75);
+  color: ${C_WHITE};
+  padding: ${GEL_MARGIN_BELOW_400PX};
+  border: none;
+`;
+
+const Content = styled.div`
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  height: ${GEL_SPACING_TRPL};
+  padding: ${GEL_SPACING_DBL} ${GEL_MARGIN_BELOW_400PX};
```

Comment:
I dont think `GEL_MARGIN_BELOW_400PX` is right here, we should use the `GEL_SPACING_` values :)